tailormap-components/projects/core/src/lib/user-interface/attributelist/attributelist-common/attributelist-datasource.ts

```diff
@@ -408,9 +513,9 @@ export class AttributeDataSource extends DataSource<any> {
 
     for (const attr of metadata.attributes) {
       // Check feature type.
-      if (attr.featureType === featureType) {
+      // if (attr.featureType === featureType) {
```

Comment:
If this `if` is not needed anymore maybe its better to just remove so we don't get confused in the future